packages/inter-protocol/src/auction/auctioneer.js

```diff
@@ -262,7 +276,12 @@ export const start = async (zcf, privateArgs, baggage) => {
 
   const publicFacet = augmentPublicFacet(
     harden({
+      /** @param {Brand<'nat'>} collateralBrand */
       getBidInvitation(collateralBrand) {
```

Comment:
should be `makeBidInvitation`, right? it returns a fresh `Invitation` each time.

Also: `mustMatch(collateralBrand, BrandShape)` seems in order.